src/main/java/aquarium/shop/order/OrderControler.java

```diff
@@ -0,0 +1,37 @@
+package aquarium.shop.order;
+
+
+import aquarium.shop.product.Product;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.math.BigDecimal;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+
+@RestController
+public class OrderControler {
+
+    @Autowired
+    private OrderRepository repository;
+
+    @GetMapping("/orders")
```

Comment:
I think we don't want to merge that insert test in such way. If we want to do this java style I think we should find another way to do such insert just after application startup (maybe Spring **postConstruct** method on some other way?) not via fake getter.